pkg/reconciler/intevents/pullsubscription/static/pullsubscription.go

```diff
@@ -68,11 +68,26 @@ func (r *Reconciler) ReconcileDeployment(ctx context.Context, ra *appsv1.Deploym
 			logging.FromContext(ctx).Error("Error propagating authentication check message", zap.Error(err))
 			return err
 		}
+		// If source is in secret mode, we need to check if the secret is absent.
+		if src.Spec.Secret != nil {
+			logging.FromContext(ctx).Error("Start to get pod list", zap.Error(err))
+			for _, pod := range podList.Items {
+				eventList, err := authcheck.GetEventList(ctx, r.KubeClientSet, pod.Name, src.Namespace)
+				logging.FromContext(ctx).Error("Start to get event list", zap.Error(err))
```

Comment:
Is this meant to be checked in?

Reply:
I think the same comment applies to line 73 as well.

Reply:
My bad, forgot to remove the debug lines after manual test...